app/scripts/views/sign_up.js

```diff
@@ -174,6 +181,16 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
         isEmailOptInVisible: this._isEmailOptInEnabled(),
         isMigration: this.isMigration()
       };
+
+      if (isSync && this.isInExperiment('syncCheckbox')) {
+        this.experimentTrigger('syncCheckbox.triggered');
```

Comment:
No, I think we should rename `experimentTrigger` to something else, `experimentSignal` or something related to https://github.com/mozilla/fxa-content-server/pull/2943#discussion_r38872463